### PR TITLE
Create draft-roadmap-2024.md

### LINKED ISSUE #133: Strategic roadmap
This is a sub-task of the 
- https://github.com/ubiquity/series-a/issues/4

### PR DESCRIPTION
**Pull request by sergfeldman**
Resolves ubiquity/research#133

### CODE REVIEW

**Comment by 0x4007** [comment]
I took a look from my phone and it seems a bit limited but it's a useful initiative for our fundraising and partnership efforts. 

Regarding deliverable format: I think it would be optimal if you tried doing all the work here in a markdown file on GitHub instead of linking back Google Docs. If you are primarily working with table data then maybe it makes sense to render CSV files and post them here. Just as long as the delivered files are in a plaintext format, then it's easy for an LLM to process it. 

It's hard to say with confidence what is the best delivery format for every type of project, but I do hope to have all the data accessible on GitHub here for when we have UbiquiBot consume the content to have better understanding of all current operations. 

If we change the delivery format to links to separate webapps, I don't see it being feasible to scale keep up with having to build adapters to access data from different websites.

I hope that this works for you.

**Comment by sergfeldman** [comment]
Yes, that makes sense.
I moved the content to the markdown file on GitHub and added an image with the structure for discussion.

File: draft-roadmap-2024.md
Comment (0x4007) [inline draft-roadmap-2024.md:1]: Looks decent but I wonder if we can make it more substantial and add more hype for marketing purposes? Perhaps for stuff later this year and early next year?

I’m excited about an XP system being implemented, and hosting hackathons to get the community to compete and build plugins (which are directly monetizable if they are useful.)

**Comment by sergfeldman** [comment]
As you recommended [here,](https://github.com/ubiquity/series-a/issues/4) I created a subtask 
- https://github.com/ubiquity/research/issues/133

File: draft-roadmap-2024.md
Comment (sergfeldman) [reply draft-roadmap-2024.md:1]: > Looks decent but I wonder if we can make it more substantial and add more hype for marketing purposes? 

Good point. I`ve added activities that can cause hype.

**Comment by sergfeldman** [comment]
@pavlovcik Can you please review the updated version of the roadmap?
@Draeieg rfc

File: draft-roadmap-2024.md [hunk 1]
```diff
@@ -1,49 +1,81 @@
...
-### Q1
+### Q1 2024
+
+Security assessment
+
+### Q2 2024
+
+Ubiquity Dollar stable version launch
+
+Ubiquity Card integration to DAO OS
```
Comment (0x4007) [inline draft-roadmap-2024.md:19]:
```suggestion
Ubiquity Card integration to Ubiquity OS
```

This might be a tough timeline but @rndquu would know best

File: draft-roadmap-2024.md [hunk 2]
```diff
@@ -1,49 +1,81 @@
...
+
+### Q3 2024
+
+DAO OS Kernel stable version launch
+
+Ubiquity Dollar integration to DAO OS
```
Comment (0x4007) [inline draft-roadmap-2024.md:25]: It might make sense to sync the card roll out with the dollar as we plan to only support functionality using our dollar.

File: draft-roadmap-2024.md [hunk 3]
```diff
@@ -1,49 +1,81 @@
...
+
+### Q4 2024
+
+DAO OS Plugins open for contribution
+
+DAO OS AI agents alpha version
```
Comment (0x4007) [inline draft-roadmap-2024.md:31]:
```suggestion

```

Replaced in favor of plugins. Technically it would just be a class of plugins. For example, we create an agent template (not a priority now) and then let the user configure the initialization prompt in plain english to configure it, for example.

File: draft-roadmap-2024.md [hunk 4]
```diff
@@ -1,49 +1,81 @@
...
+
+### Q4 2024
+
+DAO OS Plugins open for contribution
+
+DAO OS AI agents alpha version 
+
+### Q1 2025
+
+Ubiquity Bank alpha version
```
Comment (0x4007) [inline draft-roadmap-2024.md:35]: What is your definition of this? I think we are iteratively coming to this reality.

File: draft-roadmap-2024.md [hunk 5]
```diff
@@ -1,49 +1,81 @@
...
+
+DAO OS Plugins open for contribution
+
+DAO OS AI agents alpha version 
+
+### Q1 2025
+
+Ubiquity Bank alpha version 
+
+DAO OS contributors plugins launchpad
```
Comment (0x4007) [inline draft-roadmap-2024.md:37]: Can you elaborate on this? I think we could do hackathons as previously mentioned but I'm not sure what you have in mind for "launchpad"

File: draft-roadmap-2024.md [hunk 6]
```diff
@@ -1,49 +1,81 @@
...
 
-### Q2
+### Q2 2024
 
-- Ubiquity Dollar integration to DevPool
-- Ubiquity Card integration to DevPool
+DAO OS partners onboarding
 
-### Q3
+Ubiquity Dollar liquidity mining campaign
```
Comment (0x4007) [inline draft-roadmap-2024.md:55]: Starting to feel like this will be early Q3 but it is more strategic for us to do Q2.

File: draft-roadmap-2024.md [hunk 7]
```diff
@@ -1,49 +1,81 @@
...
+Ubiquity Dollar liquidity mining campaign 
 
-- AI agents for DevPool alpha version
+### Q3 2024
 
-### Q4
+DAO OS partners onboarding
 
-- Ubiquity Bank alpha version
+Series A fundraising
```
Comment (0x4007) [inline draft-roadmap-2024.md:61]: This starts next week actually. So Q2.

File: draft-roadmap-2024.md (hunk 1 above)
Comment (rndquu) [reply draft-roadmap-2024.md:19]: For me "Ubiquity Card integration to DAO" means that we should be able to issue virtual and physical cards to, let's say, 10 countries

We will be on time with issuing virtual cards for Q2 2024 for at least 1 country but physical cards and integrating more countries should be moved later

File: draft-roadmap-2024.md (hunk 4 above)
Comment (sergfeldman) [reply draft-roadmap-2024.md:35]: The roadmap usually indicates when an item is available to users (not started by us but completed).

If we are serious about becoming a Web3 bank for individuals, then it makes sense to provide standard banking services for them https://en.wikipedia.org/wiki/Bank

If by “metaverse bank” we mean something non-usual, then we need to determine what exactly it is.

I think that the alpha version of the bank should have at least secure user accounts, payment cards, and vaults.
Other services such as cashback, loans, currency exchange, and others may be provided later.

File: draft-roadmap-2024.md (hunk 5 above)
Comment (sergfeldman) [reply draft-roadmap-2024.md:37]: Ecosystem projects, based on which users can create their own products, have a higher value.
Any blockchain is a good example.
GitHub, through its app marketplace, also aims to become an ecosystem.

We discussed earlier that each plugin is a separate product.
We also discussed that external developers can create their own plugins on Ubiquity OS.
The proposed "launchpad" means that Ubiquity will facilitate launching or attract early users for the plugins.

Also, each plugin can provide an airdrop in some form - tokens, NFTs, points, etc. Airdrops can attract hype that is beneficial for Ubiquity.

File: draft-roadmap-2024.md (hunk 4 above)
Comment (0x4007) [reply draft-roadmap-2024.md:35]: Most of what you mentioned comes out of the box with DeFi. We don't really need to build anything except for some UI integrations and then the whole payment card bit. 

> If by “metaverse bank” we mean something non-usual, then we need to determine what exactly it is.

I guess I need to think on this more but the vision is that people use our system, the DevPool (or Ubiquity OS - whatever we want to call it) and any idle capital in the system we are able to make money on. This to me sounds like a bank. This also means that we should heavily incentivize partners/contributors to leave money inside of the system: hence why I think it may be presented more as a "bank" on the frontend/user facing side.

Comment (sergfeldman) [reply draft-roadmap-2024.md:35]: @pavlovcik As far as I understand, the vision you talked about is something like “earn in Metaverse, spend conveniently everywhere”:
1) Earn 
- By completing tasks
- By providing capital
- Analyze everything in your account
2) Spend
- Using a card

If so, then some functionality is missing. This could be a solution like
- Proxy Yield Aggregator you mentioned a while ago (maybe like Beefy)
- automation of LP on Uniswap which we once discussed (like Gamma)
- specialized Vaults that I offered (like PLAY)
- integration with certain DeFi protocols (like Dai Savings Rate (DSR) of MakerDao)
- direct use of DeFi pools (Ubiquity just provides a dashboard)
- or something else
